tests/data/library_of_congress/bd01x09x.xml

```diff
@@ -309,14 +309,12 @@
     <datafield tag="017" ind1=" " ind2=" ">
       <subfield code="a">99-263</subfield>
       <subfield code="b">BwMiBKP </subfield>
-      <subfield code="2"></subfield>
```

Comment:
I think the problem is related with the fact that empty string is consider as `None` 
IMHO, as I wrote before https://github.com/inveniosoftware/dojson/pull/155#discussion_r70613646, no value is a value and therefore empty strings should be preserved or at least give the opportunity to decide if they are kept or nor